processor/src/decorators/mod.rs

```diff
@@ -350,8 +413,241 @@ mod tests {
         assert_eq!(expected_stack, process.stack.trace_state());
     }
 
+    proptest! {
+        #[test]
+        fn arbitrary_smtget_opens_correctly(
+            ref seed in any::<[u8; 32]>()
+        ) {
```

Comment:
What do we get from this test here? Seems like it is very similar to the one in the `stdlib`.

Reply:
One tests only the output of the advice injector and inspect the stack, the other tests the full functionality of the MASM `smtget` implementation